app/views/users/profile_edit/_favorite_organizations.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render "users/profile_sections/profile_section", title: "FAVORITE ORGANIZATIONS"  do %>
+  <div class="user-fav-orgs-list">
+    <% @user.user_favorite_organizations[0..3].each do |user_fav_org| %> <!-- show only 4 favorite organizations for now. -->
```

Comment:
Sorry I didn't see this either. I think we should similarly add this to one of helpers and also make pulling the top `number` as a constant variable.

Also test for this as well..

Reply:
I don't think this one needs to be moved to any helper right now. What it's doing is quite minimal and it doesn't seem confusing. I can always refactor it eventually if/when more logic is added and things get more complicated.

Reply:
This applies to the same reason as my request to other helpers. These are to abstract away logic in layout to provide better readability and testability, which is something we should strive for and demonstrate as software engineers

Reply:
I’ll be happy to look more into this on a different task that more focused on this section. I think it’s okay for now.

Reply:
I’ll be happy to look more into this on a different task that more focused on this section. I think it’s okay for now.

Reply:
I'm not sure why this portion is "different". This is an implementation that relates to two trello stories you worked from, and the change request is almost identical to requests I had to something that's very similar to this. 

I would be willing to fallback if I'm really pushing a difficult refactor, but this is something that will improve the view implementation that is in this PR.

Reply:
For this one, I really don't think a helper is needed for this one. But I can remove the extra iteration logic though. I'll pass in a collection to the partial instead. I'll make the change and you let me what you think.

Reply:
> This is an implementation that relates to two trello stories you worked

Not sure what you mean.

Reply:
@sanghun89 I've removed some of the logic from the view. Take another look at it.
There's probably other places that logic can be extracted from. But I still don't think it's within the scope of the pr to extract all the view logic from all places. I'll be happy to focus on that in subsequent tasks.